### ISSUE TITLE
FBT Enums only work as `require` statements

### ISSUE THREAD

**Issue by jrwats** [facebook/fbt#46]
The following blows up because the enum isn't required via `require(...)`, but instead uses import
```
import { fbt } from 'fbt-runtime'
import CategoriesEnum from './Categories$FbtEnum'
function category(listing) {
  return (
    <span>
      <fbt desc="Category">
        <fbt:enum enum-range={CategoriesEnum} value={listing.category} />
      </fbt>
    </span>
  )
}

export default category
```

**Comment by jrwats** [comment]
As of 1/30/2019
056251d26a0f4f7b1d7610d327bcce6294f291ba:
[Relevant code](https://github.com/facebookincubator/fbt/blob/master/transform/babel-plugin-fbt/index.js#L215-L222)

**Comment by jrwats** [comment]
Oh NVMD

**Comment by jrwats** [comment]
@pieterv is on the case!

**Comment by pieterv** [comment]
Fixed via 87133a434d4db65afc0f360612033a010182807b!
